docs/src/main/sphinx/release/release-353.md

```diff
@@ -0,0 +1,87 @@
+# Release 353 (XXXXXXXX March 2021)
+
+## General
+
+* Fix aliasing of columns in common table expressions. ({issue}`6839`)
+* Fix potential incorrect query results for queries containing multiple `<` predicates. ({issue}`6896`)
+* Always show `SECURITY` clause in `SHOW CREATE VIEW`. ({issue}`6720`)
+* Fix reporting column references for aliased tables. ({issue}`6972`)
+* Fail `SHOW COLUMNS` when there are issues with getting information about table columns.
+  Previously empty list of columns was returned. ({issue}`6958`)
+* Improve parallelism of bucketed tables inserts. Inserts into bucketed tables can now be parallelized
```

Comment:
shall i move this to "Hive"?

cc @sopel39

Reply:
Yes, this is hive